src/modules/get.js

```diff
@@ -0,0 +1,7 @@
+import renderList from './display.js';
+
+const getData = () => fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/jplay/scores/')
+  .then((response) => response.json())
+  .then((data) => renderList(data.result));
```

Comment:
- [ ] Same issue here, you should use `async` and `await` 👍

Reply:
Thanks for alerting me. It just escaped me. Resolved now